sponsors.html

```diff
@@ -14,5 +26,6 @@ <h3 class="clemson-orange-text">By sponsoring, you gain ...</h3>
     <li class="collection-item black">For Diamond sponsors, the opportunity to be a keynote speaker</li>
   </ul>
   <h3 class="clemson-orange-text">How to sponsor</h3>
-  <p class="flow-text">For more information on ways to become involved with <strong>CUCTF 2020</strong>, please download our <a href="/public/files/CUCTF_Sponsorship_Packet.pdf" download="">Sponsor Packet</a> or contact us at <a href="mailto:sponsor@cuctf.io">sponsor@cuctf.io</a>.</p>
+  <p class="flow-text">For more information on ways to become involved with <strong>CUCTF 2020</strong>, please download our <a href="/public/files/CUCTF_Sponsor_Packet.pdf" download="">Sponsor Packet</a> or contact us at <a href="mailto:sponsor@cuctf.io">sponsor@cuctf.io</a>.</p>
+  <br> 
 </div>
```

Comment:
I'm not sure if this was part of a larger change that didn't make it in the PR but it doesn't do much and just makes the spacing at the bottom of the page inconsistent with the other pages. If you are wanting blank space at the end of pages, adding a global margin to the `container` class would be the right way to do that but I would avoid adding much if we do. That would be a change that can be made outside of the PR later as well.

```suggestion
  <p class="flow-text">For more information on ways to become involved with <strong>CUCTF 2020</strong>, please download our <a href="/public/files/CUCTF_Sponsor_Packet.pdf" download="">Sponsor Packet</a> or contact us at <a href="mailto:sponsor@cuctf.io">sponsor@cuctf.io</a>.</p>
</div>
```

Reply:
Will look into adding margin at bottom of pages. Reason being, usually when I read webpages, I keep the text somewhat... in the center of the page. It's a little uncomfortable to read the last couple sentences.

Reply:
Alright. I just pushed an addition to the style for `.container` that adds another 4em of padding to the bottom of the page. If https://dev.cuctf.io/ looks good to you, then if whoever merges this (probably will be me) rebases on dev before merge, the style will be applied.